### PR TITLE
Documentation update

### PR DESCRIPTION
**Pull request by juandelgado**
Good bash to the docs. 

Made README for all audiences, moved technical info to a dedicated space.

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Wordy Developer Docs
+
+## Status
+
+[![Build Status](https://travis-ci.org/ustwo/wordy-slack-bot.svg?branch=master)](https://travis-ci.org/ustwo/wordy-slack-bot)
+
+## Configuration
+
+ * `SLACK_TOKEN`: Slack Bot token. You can create a bot and get a token in https://YOURSLACK.slack.com/services/new/bot. For more information you can read [Slack Bots documentation](https://api.slack.com/bot-users).
```
Comment (arnau) [inline docs/development.md]: the URL `https://YOURSLACK.slack.com/services/new/bot` should be wrapped in backticks (`) so Github does not autolink it.

File: docs/development.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * `SLACK_COMMAND_TOKEN` and `SLACK_TEAM_ID` to validate that commands are coming from your team.
+ * `FIREBASE_PROJECT_ID`, `FIREBASE_CLIENT_EMAIL`, `FIREBASE_PRIVATE_KEY`, `FIREBASE_DB_URL` if you want to provide persitance through Firebase.
```
Comment (arnau) [inline docs/development.md]: So, if I choose not to set these four environment variables, where do the data go? memory?

**Review by arnau — CHANGES_REQUESTED** [review]

File: docs/development.md (hunk 2 above)
Comment (juandelgado) [inline docs/development.md]: Right now they are a hard requirement. Have a look at #10, definitely want to improve handling missing ENV vars.

Comment (arnau) [inline docs/development.md]: So, do you want to keep this wording that suggests you can opt-out firebase? or perhaps add a clarification until #10 is implemented?

**Review by arnau — APPROVED** [review]

**Pull request merged**